src/components/augmint-ui/bootstrapTable/index.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+// COULD use moment js
+import moment from "moment";
+import { Table, Tr, } from 'styled-table-component';
+import { StyleTd } from "./styles";
+
+export class BootstrapTable extends React.Component {
+    render() {
+        const { mockData, extraClass } = this.props;
```

Comment:
is the `extraClass` used somewhere?

Reply:
i saw it used in src/components/Table in the same position so i though that if you guys use it on a table, probably i should use it as well.